--- ii/.config/quickshell/ii/services/DateTime.qml	2025-07-29 20:55:49.710551114 +0800
+++ work/.config/quickshell/ii/services/DateTime.qml	2025-07-29 22:06:33.316688156 +0800
@@ -14,8 +14,8 @@
         precision: SystemClock.Minutes
     }
     property string time: Qt.locale().toString(clock.date, Config.options?.time.format ?? "hh:mm")
-    property string date: Qt.locale().toString(clock.date, Config.options?.time.dateFormat ?? "dddd, dd/MM")
-    property string collapsedCalendarFormat: Qt.locale().toString(clock.date, "dd MMMM yyyy")
+    property string date: Qt.locale().toString(clock.date, Config.options?.time.dateFormat ?? "dddd, MM/dd")
+    property string collapsedCalendarFormat: Qt.locale().toString(clock.date, "yyyy MMMM dd")
     property string uptime: "0h, 0m"
 
     Timer {
